src/_cffi_src/openssl/x509.py

```diff
@@ -144,6 +144,7 @@
 int X509_REVOKED_set_revocationDate(X509_REVOKED *, ASN1_TIME *);
 
 X509_CRL *X509_CRL_new(void);
+X509_CRL *X509_CRL_dup(X509_CRL *crl);
```

Comment:
Our style for bindings is to not include variable names here. Otherwise this LGTM.

Reply:
Done